support-frontend/assets/components/forms/customFields/checkbox.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React, { type Node } from 'react';
+
+import './checkbox.scss';
+
+// ----- Types ----- //
+
+type PropTypes = {
+  text: Node,
```

Comment:
Do you not need to specify appearance here?

Reply:
Good pickup @lucymonie  Ta!